conda-store-server/conda_store_server/server/views/api.py

```diff
@@ -9,11 +9,37 @@
 from fastapi import APIRouter, Body, Depends, HTTPException, Query, Request
 from fastapi.responses import PlainTextResponse, RedirectResponse
 
-router_api = APIRouter(
+_API_PREFIX_V1 = "/api/v1"
+_API_PREFIX_V2 = "/api/v2"
+_API_PREFIX_EXPERIMENTAL = "/api/experimental"
+
+router_api_v1 = APIRouter(
+    tags=["api"],
+    prefix=_API_PREFIX_V1,
+)
+
+router_api_v2 = APIRouter(
     tags=["api"],
-    prefix="/api/v1",
+    prefix=_API_PREFIX_V2,
 )
 
+router_api_experimental = APIRouter(
+    tags=["api"],
+    prefix=_API_PREFIX_EXPERIMENTAL,
+)
```

Comment:
The experimental API is not currently used, but I've added it while I'm making changes to routing, since it's part of the BC policy proposal.